king_phisher/client/graphs.py

```diff
@@ -801,3 +812,144 @@ def _check_complexity(self, password):
 					met += 1
 					break
 		return met >= 3
+
+class CampaignCompGraph(GraphBase):
+	""" Display selected campaigns data by order of campaign start date."""
+	graph_title = 'Campaign Comparison Graph'
+	name_human = 'Graph'
+	def _load_graph(self, data):
```

Comment:
This should be a public method. Also data should be a list of campaign IDs, and it should be clearly documented.